src/rest_framework_dso/fields.py

```diff
@@ -231,6 +235,18 @@ def apply(
                 code="fields",
             )
 
+        # We keep the order of the _fields parameter that has been passed in.
+        if self._ordered_fields:
+            display_fields = {}
+            for field_name in self._ordered_fields:
+                if field_name in fields_to_keep:
+                    display_fields[field_name] = fields[field_name]
+            for field_name, field in fields.items():
+                if field_name in always_keep:
+                    display_fields[field_name] = field
```

Comment:
dit zet `_links` achteraan, nietwaar? en gaat nogmaals over de hele lijst heen..
Het lijkt een beetje dubbel. misschien dit apart doen?

```suggestion
            for field_name in always_keep:
                if (field := fields.get(field_name)) is not None:
                    display_fields[field_name] = field
```